### DISCUSSION TITLE
wc花了一个多小时，终于进来了

### DISCUSSION THREAD

**Discussion by maoyesnd** [share121/inter-knot#722] [其他]
![1721543377195](https://github.com/user-attachments/assets/ab26e7d9-7331-4338-80af-b8c13d6793d3)

**Comment by maplezhi** [comment]
我也一个小时qwq

**Comment by aiTunes-dev** [comment]
恭喜，多看看大佬教程就豪

**Comment by firavoyage** [comment]
github的issues和discussion为什么一起编号...

**Comment by cheeseRatRat** [comment]
裸连确实难进，没萝卜进绳网都难

**Comment by Kaohaaa** [comment]
我被OctoCaptcha折磨好久

**Comment by Kk-k11** [comment]
这个委托我接了!

**Comment by GanlinSama** [comment]
四楼，进绳网为什么要萝卜？不是靠魔法么？

**Comment by WLMRMZ** [comment]
纳尼，你们竟然要挂萝卜，我是挂零号空洞进来的（）(￣﹃￣)

**Comment by capacitor1** [comment]
hhh还好我是资深VPN用户，Github也老早就注册了

花3分钟装了个油猴插件就进来了doge

**Comment by Buer-Nahida** [comment]
我花了30秒就进来了

**Comment by Ropenetsuser10086** [comment]
这玩意不就随便找个HUD或者手机连上萝卜网络加速再用绳网邮箱注册一下的事吗

**Comment by Buer-Nahida** [comment]
我本来就有Github和魔法，直接安装个脚本再用Github登陆下就行了

**Comment by SDCOM-0415** [comment]
我是**被处理过的**VPN老灯

**Comment by ChevalierDeSangreal** [comment]
爱门！

**Comment by lileyi441** [comment]
嗯呢嗯呢
